spec/spec.md

```diff
@@ -1040,6 +1046,17 @@ When using this [[ref:Feature]]:
   the same [[Ref:Subject]] as any other [[ref:Claims]] in the presented
   application.
   :::
+  :::note
+  The `subject_is_issuer` property can be used by [[ref:Verifiers]] as a means 
+  to prompt for user-entered data flows to be facilitated by client apps 
+  (e.g. user agent wallets). This can be conveyed to consuming apps by setting 
+  the `subject_is_issuer` property to `preferred` on the applicable 
+  [[ref:Input Descriptor]] and including whatever `fields` are desired for 
+  submission. An example use of this functionality would be to provide a 
+  fallback for the user to fill out a form that is generated based on the 
+  JSON Schema `filter` objects present in the `fields` of an [[ref:Input Descriptor]] 
+  for submission as _self-attested_ data.
```

Comment:
I think that this note is great and would be in favour of it being merged as is.

That said, for what it's worth, Energy Web and Elia Group have implemented [a service here](https://github.com/energywebfoundation/ssi/tree/develop/apps/input-descriptor-to-credential) which takes JSON Schemas Input Descriptor `filter` objects and converts them into an (unsigned) credential.
Our planned use case for this is for user consent. For example, a Presentation Definition could require a VC describing some data as well as a self-signed credential providing consent on how the data will be used.
As such, in the user consent use case, I think the `subject_is_issuer` would be `required` (rather than `preferred`) as we want the actual user to be providing consent so a non `subject_is_issuer` will not do. So I would consider amending this note slightly to indicate that both `preferred` or `required` could indicate this to the consuming app.

```suggestion
  The `subject_is_issuer` property can be used by [[ref:Verifiers]] as a means 
  to prompt for user-entered data flows to be facilitated by client apps 
  (e.g. user agent wallets). This can be conveyed to consuming apps by setting 
  the `subject_is_issuer` property to `preferred` or `required` on the applicable 
  [[ref:Input Descriptor]] and including whatever `fields` are desired for 
  submission. An example use of this functionality would be to provide a 
  fallback for the user to fill out a form that is generated based on the 
  JSON Schema `filter` objects present in the `fields` of an [[ref:Input Descriptor]] 
  for submission as _self-attested_ data.
```

Reply:
I have adjusted the language to note both can act this way